### ISSUE TITLE
Nastavení velikosti obrázku na mobilním zařízení

### ISSUE THREAD

**Issue by marekdedic** [skaut/SRS#919]
Obrázky ve stránkách SRS se zobrazují v plné šířce a nerespektují maximální šířku obrazovky na mobilních zařízeních.

**Comment by DeetsCZ** [comment]
@jde o obrázky, které jste si přidali?
Bylo by řešením si je donastavit pomocí css?

**Comment by marekdedic** [comment]
Ano, jde o námi přidané obrázky. Pomocí CSS by je šlo donastavit, nicméně to většina uživatelů nezvládne - navrhuji tohle nastavit přímo v CSS samotného SRS, přijde mi to jako lepší výchozí chování.
